### PR TITLE
[BB-2801] Fix integration tests for Juniper

### PR DESCRIPTION
**Pull request by 0x29a**
This PR fixes four issues that I've encountered during running CI using Juniper instances.

**JIRA tickets**:
- [BB-2801](https://tasks.opencraft.com/browse/BB-2801)

**Dependencies**:
https://github.com/open-craft/configuration/tree/integration-juniper

**Testing instructions**:

Passing of all CI checks is enough.

**Reviewers**
- [ ] @lgp171188

### CODE REVIEW

File: instance/tests/integration/integration_instance.py [hunk 1]
```diff
@@ -354,7 +354,7 @@ def assert_static_content_overrides_work(self, instance, appserver, page):
         homepage_html = get_url_contents(instance.url)
         self.assertIn(instance.static_content_overrides['homepage_overlay_html'], homepage_html)
 
-        page_url = '{}/{}'.format(instance.url, page)
+        page_url = instance.url + page
```
Comment (pcockwell) [inline instance/tests/integration/integration_instance.py:357]: Is there a particular reason for this change? Will the previous syntax not work with the Python version upgrades?

File: playbooks/collect_activity/collect_activity.yml [hunk 2]
```diff
@@ -22,11 +22,12 @@
       become_user: www-data
 
     - name: Run collect_activity script
-      command: >
+      shell: >
+        . /edx/app/edxapp/edxapp_env &&
```
Comment (pcockwell) [inline playbooks/collect_activity/collect_activity.yml:26]: I think `source /edx/app/edxapp_env` would be more explicit here

**Review by pcockwell — CHANGES_REQUESTED** [review]

File: instance/tests/integration/integration_instance.py (hunk 1 above)
Comment (0x29a) [reply instance/tests/integration/integration_instance.py:357]: Yes, @pcockwell, by some reason `instance.url` contains trailing slash in Juniper unlike previous release, so this was breaking test.

File: playbooks/collect_activity/collect_activity.yml (hunk 2 above)
Comment (0x29a) [reply playbooks/collect_activity/collect_activity.yml:26]: @pcockwell `shell` module uses `/bin/sh` under the hood, which has no `source` command.

**Review by pcockwell — APPROVED** [review]
Good to go assuming tests complete successfully :+1:

- [ ] I read through the code

File: instance/tests/integration/integration_instance.py (hunk 1 above)
Comment (lgp171188) [reply instance/tests/integration/integration_instance.py:357]: @0x29a, `instance.url` had a trailing slash even on Ironwood. So fixing this is good. 👍🏼

**Review by lgp171188 — APPROVED** [review]
:+1: @0x29a, this is good to merge. While it will be ideal to merge after addressing https://github.com/open-craft/configuration/commit/72144b3e0615c5f154191757f64354753f66221b#r41717195, we can address that in a separate PR as there isn't a lot of time before the sprint ends.
- [x] I tested this works by checking the integration test run.
- [x] I read through the code
- [x] ~~Includes documentation~~ NA

**Pull request merged**